neo/Network/P2P/Payloads/BlockBase.cs

```diff
@@ -32,11 +32,25 @@ public UInt256 Hash
             }
         }
 
-        public virtual int Size => sizeof(uint) + PrevHash.Size + MerkleRoot.Size + sizeof(uint) + sizeof(uint) + NextConsensus.Size + Witness.Size;
+        public virtual int Size => sizeof(uint) + PrevHash.Size + MerkleRoot.Size + sizeof(uint) + sizeof(uint) + NextConsensus.Size + 1 + Witness.Size;
+
+        Witness[] IVerifiable.Witnesses
+        {
+            get
+            {
+                return new[] { Witness };
+            }
+            set
+            {
+                if (value.Length != 1) throw new ArgumentException();
+                Witness = value[0];
```

Comment:
Rename `value`?

Reply:
value is a reserved word, is the received parameter that you need to use

Reply:
True!! Sorry about that haha